membership-attribute-service/app/services/GuardianPatronService.scala

```diff
@@ -85,29 +82,29 @@ class GuardianPatronService(
         casActivationDate = None,
         promoCode = None,
         isCancelled = subscription.isCancelled,
-        plans = CovariantNonEmptyList(
+        ratePlans = List(
           RatePlan(
```

Comment:
note that `RatePlan` (and `RatePlanCharge` below) are totally different from before.
The old ones were the high level ones and have been deleted.
The new ones are the old low level ones and have been renamed.